docs/notebooks/analogs.ipynb

```diff
@@ -6,8 +6,10 @@
    "metadata": {},
```

Comment:
<p>Is the ";" at the end put there on purpose (maybe there's some use in a Jupyter Notebook I don't know) ? Not sure</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/xclim/pull/1338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Nope, no reason it should have been there.